x-pack/plugins/reporting/server/usage/__snapshots__/reporting_usage_collector.test.ts.snap

```diff
@@ -2829,37 +2829,6 @@ Object {
     },
     "total": 4,
   },
-  "csv_searchsource_immediate": Object {
```

Comment:
I approve these being removed from the snapshot, since the export type that adds this is no longer getting registered into the ExportTypesRegistry